examples/cluster-go/step1/go.mod

```diff
@@ -1,94 +1,99 @@
 module github.com/pulumi/pulumi-eks/examples/cluster-go
 
-go 1.21
+go 1.21.12
+
+toolchain go1.23.1
 
 require (
 	github.com/pulumi/pulumi-aws/sdk/v6 v6.50.1
-	github.com/pulumi/pulumi-eks/sdk/v2 v2.0.0
-	github.com/pulumi/pulumi/sdk/v3 v3.93.0
+	github.com/pulumi/pulumi-eks/sdk/v3 v3.0.0-alpha.6
+	github.com/pulumi/pulumi/sdk/v3 v3.129.0
 )
 
+replace github.com/pulumi/pulumi-eks/sdk/v3 => ../../../sdk
```

Comment:
It would be good to avoid keeping the replaces but I can't seem to be able to run `go mod tidy` since there is no valid v3 SDK published yet. Trying replaces temporarily.